ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_network_services.schema.yml

```diff
@@ -105,6 +105,22 @@ $defs:
                 description: |
                   Nodes is required to restrict configuration of BGP neighbors to certain nodes in the network.
                   If not set the peer-group is created on devices which have a bgp_peer mapped to the corresponding peer_group.
+              address_family_ipv4:
+                type: dict
+                keys:
+                  peer_groups:
+                    type: list
```

Comment:
You are already under the peer-group item, so `address_family_ipv4` should just be a dict with `$ref: "eos_cli_config_gen#/keys/router_bgp/keys/address_family_ipv4/keys/peer_groups/items/"`

Reply:
Done!